### PR TITLE
Changes to interfaces to add new methods

### PR DESCRIPTION
**Pull request by swami-veeramani**

### CODE REVIEW

File: gobblin-test-harness/src/main/java/gobblin/test/execution/operator/SetupOperator.java [hunk 1]
```diff
@@ -1,5 +1,4 @@
-/*
- * Copyright (C) 2014-2015 LinkedIn Corp. All rights reserved.
+/* (c) 2014 LinkedIn Corp. All rights reserved.
```
Comment (suvodeep-pyne) [inline gobblin-test-harness/src/main/java/gobblin/test/execution/operator/SetupOperator.java]: I think you will need to update the copyright message

File: gobblin-test-harness/src/main/java/gobblin/test/setup/config/TestHarnessLauncher.java [hunk 2]
```diff
@@ -25,25 +22,10 @@
 public interface TestHarnessLauncher extends ConfigStepsGenerator {
 
   /**
-   *  This method will parse the config based on the input provided
-   *
-   * @param properties The configuration properties
-   */
-
-  public void configure(Properties properties);
```
Comment (suvodeep-pyne) [inline gobblin-test-harness/src/main/java/gobblin/test/setup/config/TestHarnessLauncher.java]: Are we removing the configure and prepareTest methods? How are you planning to send configuration parameters to the harness?

Comment (sahilTakiar) [inline gobblin-test-harness/src/main/java/gobblin/test/setup/config/TestHarnessLauncher.java]: We can just use the constructor to pass in the properties object.

File: gobblin-test-harness/src/main/java/gobblin/test/execution/operator/SetupOperator.java (hunk 1 above)
Comment (sahilTakiar) [inline gobblin-test-harness/src/main/java/gobblin/test/execution/operator/SetupOperator.java]: Yes, as Suvodeep mentioned, please do a merge with the latest code.

Comment (sahilTakiar) [inline gobblin-test-harness/src/main/java/gobblin/test/execution/operator/SetupOperator.java]: Check out: https://help.github.com/articles/syncing-a-fork/ for directions on how to do this.

**Pull request closed without merging**